modules/polls/types/pollTally.d.ts

```diff
@@ -13,22 +13,30 @@ type Result = {
 };
 
 export type PollTallyOption = {
-  firstChoice: BigNumber,
-  transfer: BigNumber,
-  winner: boolean,
-  eliminated: boolean
-}
+  firstChoice: BigNumber;
+  transfer: BigNumber;
+  winner: boolean;
+  eliminated: boolean;
+};
 
 export type RawPollTally = {
   winner: string | null;
   rounds: number;
   totalMkrParticipation: BigNumber;
   numVoters: number;
-  options: Record<number, PollTallyOption>
+  votesByAddress: PollTallyVote[];
```

Comment:
I'm thinking that we can introduce this as a new endpoint in a separate model.
 - /api/poll/[poll-id]/votes
Why?

Because we are using this model also in the vote poll history. Where we fetch like 50 polls. If we also have to fetch the voteByAddress for each poll is going to be a lot of info.  Also , if we split we can do async fetching of the vote stats. 

An alternative would be leaving it like this and making it an optional value in the model, so we don't have to fetch this when we are fetching a list of polls.